src/App.tsx

```diff
@@ -1,61 +1,133 @@
+import React, { ChangeEvent, FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const getUser = (userId: number): User | null => {
```

Comment:
don't forget about past checklists
```suggestion
const getUserById = (id: number): User | null => {
```